httpclient_android/http_client/src/test/java/com/backblaze/b2/client/webapihttpclient/android/http_client/B2StorageHttpClientBuilderTest.java

```diff
@@ -0,0 +1,24 @@
+package com.backblaze.b2.client.webapihttpclient.android.http_client;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class B2StorageHttpClientBuilderTest {
+
+    @Test
+    public void B2StorageHttpClientBuilder_builder() {
```

Comment:
are there any real tests we can put in here?

Reply:
Yes. of course